include/libtorrent/http_connection.hpp

```diff
@@ -36,7 +36,7 @@ POSSIBILITY OF SUCH DAMAGE.
 #ifndef TORRENT_HTTP_CONNECTION
 #define TORRENT_HTTP_CONNECTION
 
```

Comment:
if you add `#include "libtorrent/config.hpp"` here, I think it will build on the MacOS CI as well